stock_scanner/models/scanner_hardware.py

```diff
@@ -451,6 +451,8 @@ def scanner_end(self, numterm=None):
 
     @api.one
     def check_credentials(self, login, password):
+        if password == '':
```

Comment:
```suggestion
        if not password:
```

Reply:
Ok! It was to highlight the use case but I change it

Reply:
Done!